<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar:tab{ height: 30px; width:175px;}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>지점 관리</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>STARBOKS_3.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="loadbtn1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>90</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>조 회</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>130</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>지점위치</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>지점 관리</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Branch_address1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Branch_hp1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>전화번호</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changebtn1">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>정보 변경</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Branch_owner1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>190</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>점 주 명</string>
      </property>
     </widget>
     <widget class="QPushButton" name="joinbtn1">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>등 록</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deletebtn1">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>320</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>지점삭제</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>373</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>지 점 명</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Branch_name1">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>90</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>지점 조회</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>STARBOKS_3.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>150</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TOP 6</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>71</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>지점명</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>지점별 인기메뉴</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadbtn2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>100</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
     <widget class="QListWidget" name="TOP6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>160</y>
        <width>261</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="Brandch_name2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>100</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>지점별 매출 순위</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>711</width>
        <height>371</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>STARBOKS_3.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>각 지점별 매출 순위</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_list_7">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{font: 75 12pt &quot;맑은 고딕&quot;; border-radius: 5px; border:2px solid rgb(0, 112, 74);}
QComboBox:focus{border: 1px solid rgb(65, 205, 82)}</string>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>13</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>14</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>15</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>17</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>18</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>21</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>22</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>23</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>24</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>26</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>27</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>28</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>29</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>31</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_list_8">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{font: 75 12pt &quot;맑은 고딕&quot;; border-radius: 5px; border:2px solid rgb(0, 112, 74);}
QComboBox:focus{border: 1px solid rgb(65, 205, 82)}</string>
      </property>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="loadbtn3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_list_9">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>90</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{font: 75 12pt &quot;맑은 고딕&quot;; border-radius: 5px; border:2px solid rgb(0, 112, 74);}
QComboBox:focus{border: 1px solid rgb(65, 205, 82)}</string>
      </property>
      <item>
       <property name="text">
        <string>2021</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2022</string>
       </property>
      </item>
     </widget>
     <widget class="QListWidget" name="rank">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>321</width>
        <height>201</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:1px solid rgb(0, 112, 74);</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>총 매출 현황</string>
     </attribute>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 11pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>총 매출 현황</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_list_4">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{font: 75 12pt &quot;맑은 고딕&quot;; border-radius: 5px; border:2px solid rgb(0, 112, 74);}
QComboBox:focus{border: 1（1px solid rgb(65, 205, 82)}</string>
      </property>
      <item>
       <property name="text">
        <string>2021</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2022</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_list_5">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>50</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{font: 75 12pt &quot;맑은 고딕&quot;; border-radius: 5px; border:2px solid rgb(0, 112, 74);}
QComboBox:focus{border: 1px solid rgb(65, 205, 82)}</string>
      </property>
      <item>
       <property name="text">
        <string>01</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>02</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>03</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>04</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>05</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>06</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>07</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>08</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>09</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>11</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="loadbtn4">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>조회</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logoutbtn">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>320</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	font: 11pt &quot;맑은 고딕&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(0, 112, 74);
	border: 0px;
	border-radius: 8px;
}
QPushButton:hover{
	color: rgb(0, 0, 0);
	background-color: rgb(0, 255, 166);
}</string>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>loadbtn1</tabstop>
  <tabstop>Branch_address1</tabstop>
  <tabstop>Branch_hp1</tabstop>
  <tabstop>changebtn1</tabstop>
  <tabstop>Branch_owner1</tabstop>
  <tabstop>joinbtn1</tabstop>
  <tabstop>deletebtn1</tabstop>
  <tabstop>Branch_name1</tabstop>
  <tabstop>TOP6</tabstop>
  <tabstop>Brandch_name2</tabstop>
  <tabstop>rank</tabstop>
  <tabstop>logoutbtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
